src/Commands/ResourceCommand.php

```diff
@@ -118,30 +121,33 @@ public function searchRecord($query): EloquentCollection|Collection|array
             ));
     }
 
-    protected static function applyGlobalSearchAttributeConstraint(Builder $query, array $searchAttributes, string $searchQuery, bool &$isFirst): Builder
+    protected static function applyGlobalSearchAttributeConstraint(Builder $query, array $searchAttributes, string $searchQuery, bool &$isFirst, $resource): Builder
     {
-        /** @var Connection $databaseConnection */
-        $databaseConnection = $query->getConnection();
+        $model = $query->getModel();
 
-        $searchOperator = match ($databaseConnection->getDriverName()) {
-            'pgsql' => 'ilike',
-            default => 'like',
-        };
+        $isForcedCaseInsensitive = $resource::isGlobalSearchForcedCaseInsensitive();
 
+        /** @var Connection $databaseConnection */
+        $databaseConnection = $query->getConnection();
+        if ($isForcedCaseInsensitive){
+            $searchQuery = strtolower($searchQuery); 
+        }
```

Comment:
Why do we need to convert to lowercase? I compared to Filament's code and it's not there

Reply:
Otherwise it wont show when searching for 'Groep'

Reply:
The ```generate_search_column_expression``` is used to convert the field in the database to lowercase, so thats why I convert the ```$searchQuery``` to lowercase.